### PR TITLE
Journal and Tutorial week 4 to master

### PR DESCRIPTION
**Pull request by s4dmw**

### CODE REVIEW

**Comment by jeffworley** [comment]
As always you are to the point and accurate!  Nice progress. :+1:

**Comment by ericksonk** [comment]
Well planned outline, looking forward to seeing your thoughts on jQuery play out :+1:

**Comment by cmauban** [comment]
Solid outline :+1:  Your titles are so creative

**Comment by drumslayert** [comment]
"wait...what were we talking about again?", best line in tutorial! looking forward to the final. :+1:

File: creative-journal-title-week-4.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
-## Week 4 Journal Topics (flags in the sand)
+## Week 4 Journal outline
+Topic - jQuery
```
Comment (kellymurray) [inline creative-journal-title-week-4.md:2]: be sure to turn this into more prose-style blogging for tomorrow

**Pull request merged**